Containers/mastercontainer/internal.Caddyfile

```diff
@@ -0,0 +1,35 @@
+{
+	# auto_https will create redirects for https://{host}:8443 instead of https://{host}
+	# https redirects are added manually in the http://:80 block
+	auto_https off
+
+	storage file_system {
+		root /mnt/docker-aio-config/caddy/
+	}
+
+	log {
+		level ERROR
+	}
+
+	servers {
+		protocols h1 h2 h2c
+	}
+
+	on_demand_tls {
+		ask http://127.0.0.1:9877/
+	}
+
+	skip_install_trust
+}
+
+https://:8080 {
+	root * /var/www/docker-aio/php/public
+	encode
+	php_fastcgi unix//run/php.sock
+	file_server
+
+	tls {
+		on_demand
```

Comment:
Just as initial comment: I dont think we need to validate ips and domains? Or do we? In that case everything should be accepted, no?

Reply:
done

Reply:
it is required, but not for internal certs